<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="5b2dc50b-2a21-4212-8fe5-4984508175ec">
      <driver-ref>faca3ac3-ce0d-4081-9461-5a902d2c7a46</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/accr/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:mysql://127.0.0.1:3306/accreditation</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.mysql.MysqlExplainPlanProvider</provider>
      </explain-plan-config>
      <libraries>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/mysql-connector-license.txt</url>
        </library>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/mysql-connector-java-5.1.35-bin.jar</url>
        </library>
      </libraries>
    </data-source>
    <data-source source="LOCAL" name="Sqlite (Xerial) - db.sqlite3" uuid="bcac95dc-d0b0-4408-8cdc-ff13054b4a15">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>